modules/wilson_content/wilson_content.info.yml

```diff
@@ -0,0 +1,5 @@
+name: Wilson Content
+type: module
+description: 'Wilson overrides to the content editing experience.'
+package: Wilson
+core_version_requirement: ^8.8 || ^9
```

Comment:
Small thing, since this is a new module, could it be Drupal 10-ready? :D From https://www.drupal.org/node/3070687:

> Drupal 10 compatibility
> 
> For Drupal 10 compatibility, core_version_requirement is still used. Once you ensured the project is otherwise compatible with Drupal 10, specify the version constraint to allow Drupal 10 compatibility. Such as for a project compatible with both Drupal 9 and 10:
> 
> ```
> name: My Module
> type: module
> core_version_requirement: ^9 || ^10
> ```

Reply:
This will be superseded by the move to `.profile`. But I'll update the relevant part of `wilson.info.yml` because it applies to the profile compatibility.